data_steward/tools/deid_runner.sh

```diff
@@ -54,7 +54,8 @@ export GOOGLE_CLOUD_PROJECT="${APP_ID}"
 gcloud auth activate-service-account --key-file="${key_file}"
 gcloud config set project "${APP_ID}"
 
-cdr_deid="${cdr_id}_deid"
+cdr_deid="${dataset_release_tag}_deid"
+registered_cdr_deid="R{cdr_deid}"
```

Comment:
missing variable `$`
```suggestion
registered_cdr_deid="R${cdr_deid}"
```

Reply:
Updated